substrate/frame/support/test/tests/runtime_metadata.rs

```diff
@@ -102,17 +102,24 @@ mod apis {
 
 			fn wild_card(_: u32) {}
 		}
+		pub use ext::*;
```

Comment:
Usually I don't very like to use rust-syntax to derive a different meaning. But here it can be ok as we still reexport ext in the scope, and only `impl` and `use` item are allowed.

I also feel a syntax like:
```rust
external_impl!{ext, ext2}
```
can be more readable.